src/toil/server/app.py

```diff
@@ -33,6 +33,9 @@ def parser_with_server_options() -> argparse.ArgumentParser:
                         help="The host interface that the Toil server binds on. (default: '127.0.0.1').")
     parser.add_argument("--port", type=int, default=8080,
                         help="The port that the Toil server listens on. (default: 8080).")
+    parser.add_argument("--public_url", type=str, default="",
+                        help="The base URL to use for serving static files including the stdout and stderr logs of "
+                             "the workflows. (default: \"http://{host}:{port}\").")
```

Comment:
I'm inclined to stick with only one way of doing things to keep the UI simpler.  If we're adding `--public_url`, could we delete the `--host` and `--port` options?

Reply:
Good point. My intention with `--public_url` is, say for example, when we spin up multiple containers for the Toil server for the load balancer, we can’t use the host and port internal to each container for the generated URLs for output files and logs. We need a way to tell the Toil server what the user-facing host and port is.

Please let me know if this makes sense, or if there’s a better way to do this!

Reply:
@w-gao Maybe an `--internal_url` option and a `--public_url` option then?  Otherwise, I'm fine with keeping it as it is, but maybe adding a bit to the `help` descriptions for `--host` and `--port`.

Reply:
Hmm, `--internal_url` doesn't seem right to me since the Toil server doesn't have to live behind a reverse proxy (during testing, for example, `toil server --port 8000` seems more natural).

I see your point though. I'll see if I can find a way to unify these two. If not, I'll make sure to update the `help` descriptions to make the distinction clear.